askup/views.py

```diff
@@ -191,13 +191,18 @@ def get_questions_queryset(self, context):
             # Empty questions queryset if rendering the "qsets only" Qset
             queryset = []
         else:
-            queryset = Question.objects.filter(
-                qset_id=self.kwargs.get('pk')
-            ).order_by('-vote_value', 'text')
+            queryset = self.get_real_questions_queryset(self.kwargs.get('pk'))
 
         queryset = self.process_user_filter(context, queryset)
         return queryset
 
+    @staticmethod
+    def get_real_questions_queryset(qset_id):
```

Comment:
why is this method used in frame of the class? I would make it the model's method or put into utils module. Because it is used in other view L752

Reply:
Done. Moved into the utils general module.